pkg/agent/controller/networkpolicy/fqdn.go

```diff
@@ -594,38 +620,51 @@ func (f *fqdnController) runRuleSyncTracker(stopCh <-chan struct{}) {
 }
 
 // parseDNSResponse returns the FQDN, IP query result and lowest applicable TTL of a DNS response.
-func (f *fqdnController) parseDNSResponse(msg *dns.Msg) (string, map[string]net.IP, uint32, error) {
+func (f *fqdnController) parseDNSResponse(msg *dns.Msg) (string, map[string]ipWithTTL, error) {
 	if len(msg.Question) == 0 {
-		return "", nil, 0, fmt.Errorf("invalid DNS message")
+		return "", nil, fmt.Errorf("invalid DNS message")
 	}
 	fqdn := strings.ToLower(msg.Question[0].Name)
-	lowestTTL := uint32(math.MaxUint32) // a TTL must exist in the RRs
-	responseIPs := map[string]net.IP{}
+	maxConfiguredTTL := uint32(math.MaxUint32) // a TTL must exist in the RRs
```

Comment:
Is it important to calculate this variable? It seems to be used for logging only, but I don't see such log very useful.

Note that previous `lowestTTL` is a return value.

Reply:
Now I see it's not being updated, but it still seems meaningless, no TTL can be greater than MaxUint32, right?